ui/src/app/components/webcomponents/global-add.component.ts

```diff
@@ -13,6 +13,8 @@ import {UserPreferenceService} from "../../services/user-preference.service";
 })
 export class GlobalAddComponent extends BaseComponent implements OnInit {
   public versionId: Number;
+  @Output('isClicked') isClicked = new EventEmitter<boolean>();
```

Comment:
make it toggle click since, it's an emitter

Reply:
made changes